src/main/java/com/laserfiche/api/client/httphandlers/BeforeSendResult.java

```diff
@@ -1,16 +1,19 @@
 package com.laserfiche.api.client.httphandlers;
 
+/**
+ * Contains information about the HTTP request before it is sent.
+ */
 public class BeforeSendResult {
 
     /**
-     * @return Laserfiche Cloud regional domain.
```

Comment:
I m not sure I get it why we are removing these javadoc annotations?
I see the short description does not replace the @return annotation, although they are usually similar. The same for @param

![image](https://user-images.githubusercontent.com/102711477/207431401-46dabbeb-2c8b-4ac0-a793-1b04f475cc91.png)

Reply:
If we only use `@return` like how it was originally, then it will not display a description in the method summary, but it will display the `Returns:` in the method detail.
https://s01.oss.sonatype.org/service/local/repositories/releases/archive/com/laserfiche/lf-api-client-core/2.0.1/lf-api-client-core-2.0.1-javadoc.jar/!/index.html
![image](https://user-images.githubusercontent.com/102689343/207433308-c8bbacd1-b4d5-46de-9c84-9c5512214383.png)

If we remove the `@return` like in the PR, and add a description then it will look like this.
![image](https://user-images.githubusercontent.com/102689343/207433583-9ec36422-ffa4-4117-ad6a-505bc1b02f87.png)

Overall, I thought we don't lose any information by removing the `@returns` other than the `Returns:` keyword, but we gain a better method summary. There is also another option of duplicating the information in the description and the `@returns` so that both are populated, but thought since these are just getters and setters, it was ok to keep it simple

Reply:
yeah, I was thinking of using both the short summary and the @return annotation.
But I know that it requires a lot of manual work. So, I am ok with keeping it as is.